app/src/main/java/app/revanced/integrations/patches/VideoQualityPatch.java

```diff
@@ -21,10 +21,16 @@ public class VideoQualityPatch {
     public static void changeDefaultQuality(int defaultQuality) {
         Context context = ReVancedUtils.getContext();
         if (isConnectedWifi(context)) {
-            SettingsEnum.PREFERRED_RESOLUTION_WIFI.setValue(defaultQuality);
+            SharedPreferences wifi = context.getSharedPreferences("revanced_prefs", 0);
+            SharedPreferences.Editor wifieditor = wifi.edit();
+            wifieditor.putInt("wifi_quality",defaultQuality);
+            wifieditor.apply();
```

Comment:
Please open an issue for this, as this is temporal

Reply:
Opened one.